scripts/verify_tla_bridge_coverage.py

```
@@ -0,0 +1,204 @@
+    hits: list[BridgeHit] = []
+    for path in glob.glob(os.path.join(crates_dir, "**", "*.rs"), recursive=True):
+        try:
+            with open(path) as f:
+                content = f.read()
+        except UnicodeDecodeError:
+            continue
+        relpath = os.path.relpath(path)
+        doc_buf: list[str] = []
+        for line in content.splitlines():
+            stripped = line.strip()
+            if stripped.startswith("///") or stripped.startswith("//!"):
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)